dhis-2/dhis-services/dhis-service-analytics/src/main/java/org/hisp/dhis/analytics/table/model/AnalyticsTableColumn.java

```diff
@@ -101,6 +104,11 @@ public boolean isSkipIndex() {
     return Skip.SKIP == skipIndex;
   }
 
+  /** Indicates whether the column type is set to a non-default value. */
+  public boolean isDynamicColumn() {
+    return AnalyticsColumnType.DYNAMIC == columnType;
+  }
+
```

Comment:
If you want to check for static, it would be nice to also have a method named `isStatic`

Reply:
done